examples/lujain_work_area/Contact_Book_OOP/app_user.py

```diff
@@ -0,0 +1,39 @@
+# app_user.py
+
+from contact_book import ContactBook
+
+class AppUser:
+    def __init__(self, username, email):
+        self.username = username
+        self.email = email
+        self.contact_book = ContactBook()
+
+    def add_contact(self, contact):
+        """Add a contact to the user's ContactBook."""
+        self.contact_book.add_contact(contact)
+
+    def delete_contact(self, identifier):
+        """Delete a contact from the user's ContactBook."""
+        self.contact_book.delete_contact(identifier)
+
+    def update_contact(self, identifier, new_contact):
+        """Update a contact in the user's ContactBook."""
+        self.contact_book.update_contact(identifier, new_contact)
+
+    def search_contact(self, identifier):
+        """Search for a contact in the user's ContactBook."""
+        return self.contact_book.search_contact(identifier)
+
+    def save_contact_book(self, filename, format):
+        """Save the user's ContactBook to a file."""
+        if format == 'csv':
+            self.contact_book.save_to_csv(filename)
+        elif format == 'json':
+            self.contact_book.save_to_json(filename)
+
+    def load_contact_book(self, filename, format):
+        """Load the user's ContactBook from a file."""
+        if format == 'csv':
```

Comment:
We can just call load and not worry about, if its csv or json.

We have to point to the right Object in __init__, we can just put conditions there and then we would not need to use conditions anywhere else if we want to load or save